src/EnergyPlus/PhotovoltaicThermalCollectors.hh

```diff
@@ -118,21 +160,30 @@ namespace PhotovoltaicThermalCollectors {
         }
     };
 
+    enum class PVTModelTypes
+    {
+        Invalid = -1,
+        SimplePVTmodel = 1001,
```

Comment:
No need to use the word `PVTmodel` since it is already in the `Type` name and will have to be used as `PVTModelType::SimplePVTmodel`. `Simple` is sufficient. Do these absolutely have to be 1001 and 1002 for some reason?

Reply:
updated. not sure yet about the 1001. It was already in the code for 'Simple' model and I used 1002 for 'BIPVT'